lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2842,4 +2842,161 @@ describe('AwsCompileFunctions #2', () => {
         );
     });
   });
+
+  describe('when using fileSystemConfigs', () => {
+    it('should apply changes when single fileSystemConfig provided', () => {
+      const arn =
+        'arn:aws:elasticfilesystem:region:111111111111:access-point/fsap-a1a1a1a1a1a1a1a1a';
+      const localMountPath = '/mnt/path';
+      runServerless({
```

Comment:
For good tests efficiency, it'll be nice to test all _success_ scenarios with one `runServerless` run, as e.g. we do in case of ALB health check here: https://github.com/serverless/serverless/blob/04d463f4e438d02fef958fce34677a06645fee82/lib/plugins/aws/package/compile/events/alb/lib/healthCheck.test.js#L9-L113

Also same way I'd rather reuse the `function` fixture. At least when we can easily create a case purely by extension (and we do not need to reuse it in other tests files) it's good to keep fixtures count minimal

Reply:
makes perfect sense, I will adjust it :+1: